jak-na-konflikty.md

```diff
@@ -0,0 +1,102 @@
+
+Pokud kolega nechce řešit konflikt s tebou o samotě, vyhledejte rovnou mediátora.
+
+## Konflikt ve skupině
+
+Může se stát, že má s kolegou konflikt celá skupina. Tato situace je velmi složitá ke korektnímu řešení a zejména velmi nekomfortní pro konfrontovaného kolegu. Měl by si se proto držet těchto pravidel:
+
+* Pokud máš s kolegou osobní konflikt, nezatahuj do něj další kolegy
+* Neřešte kolegovi věci za zády
+* Ber ohled na konfrontovaného, pokud budete řešit konflikt pod nátlakem, může se stát, že bude řešení neefektivní.
+
+Vyvolání komnfliktu za skupinu by měl dělat jeden zástupce a měl by kolegovi zvědomit, že se jedná a skupinový konflikt.
+
+V případě skupinového konfliktu je potřeba začínat řešení přímo s mediátorem.
+
+## Řešení konfliktu
+
+Konflikt řeš někde, kde budete mít soukromí a když na to budete mít oba dostatečné množství času.
+
+Postupuj volně dle následujícíh bodů. Popros kolegu aby ti naslouchal a nepřerušoval tě.
+
+1. **Tohle se stalo / Tohle se děje** => Popiš bez emocí předmět konfliktu. 
+2. **Takhle se kolem toho cítím** => Popiš jaký má na tebe předmět konfliktu emocionální dopad. Pokus se svoje emoce regulovat tak, aby to ten druhý ustál.
+3. **Proč si myslím, že to bylo/je špatně** =>  Zvědom, proč podle tebe bylo jednání tvého kolegy špatně. (pokud to není absolutně zřejmé)
+4. **Potřeboval bych, aby se tohle stalo / změnilo** => Popiš, jakou očekáváš u kolegy změnu v jeho chování/jeho přístupu.
+
+Teď je řada na tobě abys kladl otázky ty, nepřerušoval a naslouchal tvému kolegovi. Upozaď ego a snaž se kolegu pochopit.
+
+1. **Co se podle tebe stalo? / Co se děje?** => Zeptej se kolegy jak předmět konfliktu vidí on. 
+2. **Jak se kolem toho cítíš?** => Zeptej se kolegy, jaké má kolem celé situace emoce.
+3. **Pokud se mnou neshouhlasíš, proč si myslíš, že to bylo/je dobře?** => Zeptej se kolegy, proč si myslí, že postupoval dobře? Snaž se do něj vcítit a jeho motivaci pochopit.
+4. **Co ode mě potřebuješ, aby se to stalo / změnilo** => Zeptej se kolegy, co potřebuje od tebe, aby ti byl schopný vyhovět.
+
+Společně se domluvte na dalším postupu.
+
+Vyjádři svůj respekt a kolegovi poděkuj za ochotu a odvahu konflikt otevřeně a upřímně řešit.
+
+Nezapomínej, že konflikt je soukromá věc. Tato schůzka je důvěrná, pokud se s kolegou nedohodnete jinak.
+
+## Když se nedaří najít řešení
+
+Pokud se vám nedaří domluvit se na dalším postupu, který by byl pro vás oba přijatelný, je na čase přizvat do řešení mediátora. 
+
+"Vypadá to, že se nám samotným nedaří shonout na dalším postupu. Kdo by nám mohl pomoci najít řešení? Komu oba dva důvěřujeme?"
+
+Ideální mediátor je:
+* Důvěryhodný pro obě strany
+* Nezaujatý v daném konfliktu
+
+Za mediátorem zajďete dohromady a poproste ho o mediaci společně. 
+
+## Role mediátora
+
+Pokud tě někdo poprosí, abys byl mediátorem konfliktu, předně jim poděkuj, že ti svěřují tak důležitou a soukromou záležitost. Mediaci přijmi pouze, pokud se na to cítíš. 
```

Comment:
@hauseja3 @Vratislav navrhuju sem přidat:

Pokud budeš mít v průběhu mediace konfliktu pocit, že už na ni nestačíš, neboj se o tom oběma stranám říct a požádat je, aby si našli nového mediátora.

(ad první thread v tomto PR)

Reply:
Souhlas

Reply:
Jestli to nepovede k nekonečným kolečkům s různými mediátory. Nebylo by to lepší rovnou eskalovat na radu?

Reply:
@Vratislav ono já trochu očekávám, že mediátor bude dost často někdo z rady starších, takže jim to nějak řekne. Ale klidně to napišme explicitně, jako "aby si našli nového mediátora a nebo se rovnou obrátili na radu starších".

Reply:
já jen doufám, že se nestane to, že se konflikt povleče dlouhý čas bez eskalace. Použil jsem tvoji formulaci, ale musíme si připustit variantu, že to budeme muset jednoho dne změnit, dle toho co ukáže prace.